src/Pod.ts

```diff
@@ -217,7 +217,11 @@ export default class Pod {
       case 'active':
         params = { nonce };
         break;
-      case 'executed':
+      case 'passed':
```

Comment:
Can you keep the 'executed' case for now? I think we might actually want to remove the 'passed' and 'rejected' cases for now as well